modules/core/portal-bootstrap/system.packages.extra.bnd

```diff
@@ -117,73 +117,73 @@ Export-Package:\
 	com.sun.syndication.io;version='1.0',\
 	com.sun.syndication.io.impl;version='1.0',\
 	\
-	com.thoughtworks.xstream.*;version='1.4.11.1',\
+	com.thoughtworks.xstream.*;version='1.4.11',\
 	\
-	java_cup.runtime.*;version='2.7.1',\
+	java_cup.runtime.*;version='2.7.2',\
 	\
 	javax.jms.*;version='1.1',\
 	\
-	javax.jws;version='1.1',\
-	javax.jws.soap;version='1.1',\
+	javax.jws;version='1.1.0',\
+	javax.jws.soap;version='1.1.0',\
 	\
 	javax.mail.*;version='1.4',\
 	\
-	javax.persistence.*;version='2.0',\
+	javax.persistence.*;version='2.1.1',\
 	\
 	javax.portlet.*;version='2.0',\
 	\
-	javax.rmi;version='1.0.5.Final',\
+	javax.rmi;version='1.0.5',\
 	\
-	javax.servlet.*;version='3.1.0',\
+	javax.servlet.*;version='1.2',\
```

Comment:
hmm, this change may break lots of things. The issue might be that the spec version doesn't match the semantic version of the packages based on the dependency we use. Maybe we need to change the dependency we use?

Reply:
I'm not sure there is something wrong with the logic, but I just checked that `lib/development/servlet-api.jar` has:
```
Export-Package: javax.servlet;uses:="javax.servlet.descriptor,javax.se
 rvlet.annotation";version="3.1.0",javax.servlet.descriptor;version="3
 .1.0",javax.servlet.annotation;uses:="javax.servlet";version="3.1.0",
 javax.servlet.http;uses:="javax.servlet";version="3.1.0"
```
so I'm not sure where the new package version above is coming from!